libs/sections/src/header/vl-header.section.ts

```diff
@@ -85,13 +89,54 @@ export class VlHeader extends BaseLitElement {
         return document.querySelector('#header__container');
     }
 
+    private get headerContainerSkeleton(): Element | null {
+        return document.querySelector('#header__container__skeleton');
+    }
+
     private injectHeaderContainer(): void {
         const vlBody = document.querySelector('[is="vl-body"]');
+        const headerContainerCssContent = `
+        #header__container {
+            min-height: 43px;
+        }
+    `;
 
         (vlBody || document.body).insertAdjacentHTML(
             'afterbegin',
             '<div id="header__container"><div id="header"></div></div>'
         );
+
+        this.addStylesToInjectedElement('#header__container', headerContainerCssContent);
+    }
+
+    private injectHeaderContainerSkeleton(): void {
+        const headerContainer = this.headerContainer;
+        const headerContainerSkeletonCssContent = `
+        #header__container__skeleton {
+            content: '';
+            height: 43px;
+            width: 100%;
+            display: block;
+            background: #FFF;
+        }
+        #header__container ~ #header__container__skeleton {
+            display: none;
+        }
+    `;
+
```

Comment:
- custom css hoort in een apart 'uig.css.ts' bestand te staan - zie bij de andere componenten
- css hoort de BEM conventie te volgen

Reply:
opgelost - `vl-header.section.uig-css.ts` gecreerd